combinations.go

```diff
@@ -0,0 +1,38 @@
+package goker
+
+import "fmt"
+
+func combinations(choose int, cards CardSet) []CardSet {
```

Comment:
This function wants a unit test.

Reply:
It had unit tests initially, but after I wrote `PossibleHands` I made this func unexported cause it seemed like I was testing implementation instead of interface. I test it indirectly in `make_hands_test.go`.